automation/jenkins_build.sh

```diff
@@ -218,22 +218,29 @@ if [ "$deploy" == "yes" ]; then
 		-e SLUG="$SLUG" \
 		-e BUILD_VERSION="$S3_VERSION_HOSTOS" \
 		-e DEVELOPMENT_IMAGE="$DEVELOPMENT_IMAGE" \
+		-e DEPLOYER_UID=$(id -u) \
```

Comment:
I think you can just do `--user="$(id -u):$(id -g)` rather than having to manually create/su to the user: https://docs.docker.com/engine/reference/run/#user